frontend/modules/sequences/sequences.helper.ts

```diff
@@ -39,14 +39,14 @@ export const getSelectedFrame = (frames, endingDate) => {
 	}
 
 	if (frames[rightMargin].dateTime <= endingDate) {
-		return frames[rightMargin];
+		return { frame: frames[rightMargin], index: rightMargin };
```

Comment:
What if we rename this function to getSelectedFrameIndex?

replace this one with
``export const getSelectedFrame = (frames, endingDate) => frames[getSelectedFrameIndex(frames, endingDate)] ;``

And use ``getSelectedFrameIndex(....)`` in ``selectNextKeyFramesDates`` ?